### PR TITLE
[WIP] bootstrap5 + icons

### PR DESCRIPTION
**Pull request by delijati**
Changes to work with bootstrap5 and bootstrap icons

### CODE REVIEW

**Comment by delijati** [comment]
We should also remove support for python < 3.10!?

**Comment by delijati** [comment]
Here a preview (will turn it off at some point) http://168.119.117.235:8523/

File: setup.py [hunk 1]
```diff
@@ -26,15 +24,12 @@ def readfile(name):
 CHANGES = readfile("CHANGES.txt")
 VERSION = '3.0.0.dev0'
 
-PY37MIN = sys.version_info[0] == 3 and sys.version_info[1] >= 7
-
 requires = [
     "Babel",
     "deform >= 2.0.15.dev0",  # .dev0 allows pre-releases.
     "pyramid >= 1.5a1",  # route_name argument to resource_url
```
Comment (stevepiercy) [inline setup.py:30]: By virtue of our Python support, this should be:

```suggestion
    "pyramid >= 2.0a0",  # Drop Python 3.5 support
```

See https://docs.pylonsproject.org/projects/pyramid/en/main/changes.html#id3

**Review by stevepiercy — CHANGES_REQUESTED** [review]
This is great. After I release deform 3.0.0a0, we will need to come back to this and bump that requirement. Thank you!

**Comment by stevepiercy** [comment]
This is awesome work. Thank you so much for your effort.

Pylons Project projects try to support currently supported versions of Python when the project is released. Right now [Python 3.8-3.12 are supported](https://devguide.python.org/versions/).

Would you please do two housekeeping tasks before I merge?

- Sign https://github.com/Pylons/deformdemo/blob/main/CONTRIBUTORS.txt
- Update the readme with the supported Python versions. https://github.com/Pylons/deformdemo#deform-demo

File: README.rst
Comment (stevepiercy) [inline README.rst:1]: I missed this. These need to be updated to drop 3.6 and 3.7, and add 3.10, 3.11, and 3.12.

Same for the trove classifiers in `setup.py`.

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Python version support

File: .github/workflows/ci-tests.yml [hunk 2]
```diff
@@ -59,7 +60,7 @@ jobs:
     - name: Setup python
       uses: actions/setup-python@v2
```
Comment (stevepiercy) [inline .github/workflows/ci-tests.yml:61]:
```suggestion
      uses: actions/setup-python@v4
```

File: README.rst [hunk 3]
```diff
@@ -6,7 +6,7 @@ Deform Demo
 
 Demonstration application for the `Deform <https://docs.pylonsproject.org/projects/deform/en/latest>`_ Python HTML form library.
 
-This application is tested on Python versions 3.6, 3.7, 3.8, and 3.9, and PyPy3.
+This application is tested on Python versions >= 3.8 <= 3.12, and PyPy3.
```
Comment (stevepiercy) [inline README.rst:9]: We should write out versions in narrative text.

```suggestion
This application is tested on Python versions 3.8, 3.9, 3.10, 3.11, and 3.12, and PyPy3.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
I did a more thorough review on this pass. A couple of minor changes, and I think it's good to go. Thank you!

**Pull request merged**